clusterman/aws/spot_fleet_resource_group.py

```diff
@@ -145,7 +145,7 @@ def _target_capacity(self) -> float:
         return self._configuration["SpotFleetRequestConfig"]["TargetCapacity"]
 
     @classmethod
-    def load(
+    def load(  # type: ignore
```

Comment:
it'd be nice to add a comment to these type ignores in case someone wants to take the effort to remove them in the future (or in case the typing failures are linked to an existing mypy issue), but that might just be a luisp-strong-opinion

Reply:
I'd love to, if only mypy would give me some better information about why it gets angry at this method signature. I tried all the permutations of removing the kwargs and turning the result type to a more generic type, but it still just complained with a "method signature doesn't match".